README.markdown

```diff
@@ -186,6 +186,20 @@ nvm uninstall
 
 	$ nvm uninstall [tab][tab]
 	my_alias        default        v0.6.21        v0.8.26       v0.10.28
+	
+## Compatibility Issues
+Nvm will encounter some issues if you have some non default settings set.
+The following are known to cause issues:
+
+Inside .npmrc:
```

Comment:
Please use `~/.npmrc` so the path is there, and the wrapping backticks produce the correct markdown syntax.